Gems/PhysX/Code/Tests/PhysXSystemTests.cpp

```diff
@@ -88,7 +99,7 @@ namespace PhysX
     TEST_F(PhysXSystemFixture, AddScenes_createsAllScenesRequested)
     {
         auto* physicsSystem = AZ::Interface<AzPhysics::SystemInterface>::Get();
-        EXPECT_TRUE(physicsSystem->GetAllScenes().size() == 0); //there should be no scenes currently created
+        EXPECT_TRUE(GetNumScenesInSystem() == 0); //there should be no scenes currently created
```

Comment:
Btw, if this fails, it will just print out that the value is false when it should be true. I think if you do something like EXPECT_EQ(GetNumScenesInSystem(), 0) it will give more info on how many scenes there actually are.

Reply:
yep, no flaky tests with shuffle after this fix.
good idea on making it EXPECT_EQ, i'll change!